docs/gallery/interactive/pyfar_coordinates.ipynb

```diff
@@ -0,0 +1,432 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "index_out, distance = c.find_nearest(find, k=3)\n",
+    "c.show(index_out[1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the distance to the finding points is also returned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "distance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Of course you can also get a copy of the points, if want to continue working just with the subset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p_1 = c[index_out[1]]\n",
+    "p_1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Find within\n",
+    "Another option is to find all points in a certain distance. Different distance measures are available, see the [pyfar documentation](https://pyfar.readthedocs.io/en/latest/classes/pyfar.coordinates.html#pyfar.classes.coordinates.Coordinates.find_within) for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Find all points within a distance of 3 from the point find\n",
+    "index_out = c.find_within(find, distance=3, distance_measure='euclidean')\n",
+    "c.show(index_out)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Logical operations with attributes\n",
+    "\n",
+    "Another way is to apply logical operations directly to the coordinate attributes.\n",
+    "As an example, we create a half arc. Then to obtain all points within a between 30° and 60° elevation, we just need to apply logical operations as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c2 = pf.Coordinates.from_spherical_colatitude(0, np.arange(0, 181, 10)*np.pi/180, 1)\n",
+    "index_out = (c2.elevation*180/np.pi >= 30) & (c2.elevation*180/np.pi <= 60)  \n",
+    "c2.show(np.where(index_out))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rotating coordinates\n",
+    "\n",
+    "You can apply rotations using quaternions, rotation vectors/matrixes and euler angles with  `c.rotate()`, which is a wrapper for [scipy.spatial.transform.Rotation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html). For example, rotating around the y-axis by 45 degrees can be done with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c.rotate('y', 45)\n",
+    "c.show()"
```

Comment:
Why is the default in degrees here? That's inconsistent with our convention of always using radians. 
Should we fix this in pyfar?